cmd/snapctl/flags.go

```diff
@@ -129,7 +125,7 @@ var (
 	flTaskMaxFailures = cli.IntFlag{
 		Name:  "max-failures",
 		Usage: "The number of consecutive failures before snap disable the task (Default 10 consective failures)",
-		Value: DefaultMaxFailures,
+		Value: -1,
```

Comment:
I see your point...I'll remove the reference to the default value from the CLI usage...Cheers!